Dialog/TilesetEditDialog.cpp

```diff
@@ -1184,6 +1230,31 @@ void TilesetEditDialog::on_pushButton_ImportPalette_clicked()
             tilesetEditParams->newTileset->SetColor(SelectedPaletteId, j, newcolor.rgba());
         }
     }
+
+    else if (selectedfilter.compare("Raw Binary palette (*.bin)") == 0)
+    {
+         QByteArray tmppalettedata;
+         QFile palbinfile(qFilePath);
+         if(!palbinfile.open(QIODevice::ReadOnly))
+         {
+             QMessageBox::critical(this, QString("Error"), QString("Cannot open file!"));
+             return;
+         }
+         tmppalettedata = palbinfile.readAll();
```

Comment:
Can we perhaps also print a warning to the console if the palette file is not exactly 32 bytes in size? I see there are protections in place to prevent crashing but we should also warn the user

Reply:
Which console exactly ? std::cout ? Or the one in OutputDockWidget ? If it's the one in OutputDockWidget how can I get the object in a clean way.

Reply:
Under the includes put:
`extern WL4EditorWindow *singleton;`
Usage in code:
`singleton->GetOutputWidgetPtr()->PrintString(QString(tr("Internal error: blah blah blah data: ")) + somedata);`

Reply:
:heavy_check_mark: